### PR TITLE
Changed site layout, extension standards and go coordinates

### PR DESCRIPTION
**Pull request by mathijsclaassen**
Hi Adrian,

It's been a while:) I've updated your lovely extension so that it works with the new site layout (and new extension standards).

I've also added the option to show go coordinates.

Please take a look and, if you don't mind, update the extension. Oh, and stop by for a game or two! (-:

### CODE REVIEW

File: background.html [hunk 1]
```diff
@@ -1,49 +1,10 @@
 <html>
```
Comment (mathijsclaassen) [inline background.html]: No inline js allowed anymore...

**Comment by apetresc** [comment]
Wow, thanks! I was also just thinking of reviving this extension since I've started using LittleGolem again :)

I will take a look at this tonight!

**Comment by apetresc** [comment]
Thank you so much for bringing this extension up to date :) My initial testing so far looks good! I'm going to continue doing some manual testing for a day or two, and add a chess-related feature I've wanted for a while, and then I will merge and update on the Google Web Store :)

Cheers!

**Comment by mathijsclaassen** [comment]
My pleasure! I'm sure there will be more enhancements in the future.

**Comment by apetresc** [comment]
Changes have been pushed to the Web Store, should appear within the next 60 minutes :) I credited your name in the store description for the coordinates feature - let me know if you would prefer I remove it!

**Comment by mathijsclaassen** [comment]
>I credited your name in the store description for the coordinates feature

Thank you. I'm flattered! And I learned a few things about chrome extension
on the way as well :-)

**Pull request merged**